client/src/components/session-timer/countdown-timer-display.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="countdown" :class="countdownClass">
+    <div class="countdown-number" :style="countdownSecondsStyle"><small class="countdown-number-text">{{formattedCountdownNumber}}</small></div>
+    <svg>
+      <circle :style="countdownCircleStyle" style="stroke: #003366" r="18" cx="20" cy="20" />
```

Comment:
This is cool :) but why didn't you use the Vuetify progress component? Eg,
https://vuetifyjs.com/en/components/progress-circular/#rotate

Reply:
I wanted something to look specifically the way I wanted to, and to keep it lightweight, CSS only. Stroke is an easy property to work with.

Reply:
Sounds good.